test/screens/v2/candidate_generator/widgets/reconstructed_alert_test.exs

```diff
@@ -213,7 +226,8 @@ defmodule Screens.V2.CandidateGenerator.Widgets.ReconstructedAlertTest do
                  now,
                  fetch_routes_by_stop_fn,
                  fetch_parent_station_sequences_through_stop_fn,
-                 x_fetch_alerts_fn
+                 x_fetch_alerts_fn,
+                 get_stations_fn
```

Comment:
Looks like one additional assertion is needed  after this one, now that there's another query that can fail:
```ex
assert [] ==
         reconstructed_alert_instances(
           config,
           now,
           fetch_routes_by_stop_fn,
           fetch_parent_station_sequences_through_stop_fn,
           fetch_alerts_fn,
           x_get_stations_fn
         )
```

Reply:
I don't think the whole thing should fail if this query fails, though. I could make sure it fails gracefully though